ovs/lib/alba.py

```diff
@@ -1440,7 +1440,6 @@ def calculate_safety(alba_backend_guid, removal_osd_ids):
         for osd in alba_backend.osds:
             if osd.osd_id in removal_osd_ids or osd.osd_id in error_disks:
                 extra_parameters.append('--long-id={0}'.format(osd.osd_id))
-        safety_data = []
```

Comment:
Why remove this?  `safety_data` can potentially be undefined on line 1460 now

Reply:
No, it will always be filled in.
The loop will keep retrying until safety_data is actually filled in or it will raise out of it, not reaching the code on line 1460